<launch>

<!-- Define parameters -->
<arg name="target_color" default="blue"/>
<arg name="behavior_tree_type" default="queue"/>
<param name="target_color" value="$(arg target_color)"/>
<param name="behavior_tree_type" value="$(arg behavior_tree_type)"/>

<!-- Run autonomy stack -->
<node pkg="tb3_autonomy" type="autonomy_node" name="autonomy_node" output="screen"/>

<!-- Run behavior tree visualization -->
<node pkg="groot" type="Groot" name="groot" output="screen" args="--mode monitor"/>

</launch>